app/src/main/java/com/fstyle/structure_android/data/repository/UserRepository.java

```diff
@@ -8,8 +8,11 @@
 import com.fstyle.structure_android.data.source.remote.UserRemoteDataSource;
 import com.fstyle.structure_android.data.source.UserDataSource;
 import io.reactivex.Completable;
+import io.reactivex.Flowable;
 import io.reactivex.Maybe;
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Optional;
```

Comment:
ERROR: Unused import: java.util.Optional. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports